src/modules/auth.ts

```diff
@@ -26,11 +26,24 @@ export const authService = new Elysia({ prefix: "/auth" }).use(authModel).post(
         }
 
         // TODO: FETCH BAAN FROM SOMEWHERE & VALIDATE USER ROLE
```

Comment:
- Remove finish todo
- I think you can map baan from baan_number to baan_name with a constant object like ["baan1_name", "baan2_name", ...]. There is no need to make another model just for Baan. Also, you can map on the frontend side.